.github/workflows/dockerprune.yml

```diff
@@ -143,5 +143,5 @@ jobs:
         env:
           DAYS: "100"
           REPO_NAME: ${{ steps.string_repo.outputs.lowercase }}
-          OWNER: ${{ steps.string_user.outputs.lowercase }}
```

Comment:
Probably should just leave the prune script alone it's not needed for building or anything. It's used for cleaning up old images from dockerhub